packages/marketplace/package.json

```diff
@@ -79,7 +79,8 @@
     "@openzeppelin/contracts": "^4.9.2",
     "@openzeppelin/contracts-upgradeable": "^4.9.2",
     "@openzeppelin/hardhat-upgrades": "^2.2.1",
```

Comment:
What's the implication here?

Reply:
When you use the marketplace package as a dependency in another place (like deploy package) the devDependencies are not installed which is nice (because you don't need them).